src/LiveDevelopment/Agents/RemoteFunctions.js

```diff
@@ -268,21 +269,15 @@ function RemoteFunctions(experimental) {
                 "box-shadow": "0 0 1px #fff",
                 "box-sizing": "border-box"
             };
+            
+            var mergedStyles = Object.assign({}, stylesToSet,  config.remoteHighlight.stylesToSet);
```

Comment:
Those are remote functions, so I cannot use `_.assign` there I guess until i inject it to the browser. Am I right? 

I've left it as `Object.assign`, just for now - until I know what exactly am I supposed to do.